app/mailboxes/imap/imap_mailbox.rb

```diff
@@ -0,0 +1,76 @@
+class Imap::ImapMailbox
+  include MailboxHelper
+  attr_accessor :channel, :account, :inbox, :conversation, :processed_mail
+
+  def process(mail, channel)
+    @inbound_mail = mail
```

Comment:
lets add a check here to prevent the email loop for chatwoot notification emails: 

ref: https://github.com/chatwoot/chatwoot/blob/e5a80195ab5516e4c3f4a55b8fb72c32338d2e0d/app/mailboxes/support_mailbox.rb#L10

Reply:
Done